src/WCSimEventAction.cc

```diff
@@ -1308,7 +1312,7 @@ void WCSimEventAction::FillRootEvent(G4int event_id,
       } else if (primaryList.count(trj->GetParentID()) ) {
 	parentType = 1;
       } else {  // no identified parent, but not a primary
-	parentType = 999;
+	parentType = trj->GetParentID();
```

Comment:
this is particle PID but not parent ID

Reply:
Yes the `parentType` variable is meant to be the parent PID. The parent ID is saved in `idPrnt`